### DISCUSSION TITLE
graphblas.matrix_multiply_reduce_to_scalar_generic doesn't take a mask_complement attribute

### DISCUSSION THREAD

**Discussion by paul-tqh-nguyen** [metagraph-dev/mlir-graphblas#192] [Q&A]
`graphblas.matrix_multiply_reduce_to_scalar_generic` doesn't take a `mask_complement` attribute, but `graphblas.matrix_multiply_generic` does. Is there any particular reason for this?

CC @seibert

**Comment by jim22k** [comment]
Because I forgot to add it when I updated `matrix_multiply` with a `mask_complement`. It should exist, because the mask/mask complement affects the reduction.
